include/math/Tensor.hpp

```diff
@@ -53,7 +50,7 @@ class Tensor : public Eigen::Tensor<_Scalar, _Rank> {
      *  CONSTRUCTORS
      */
 
-    using Eigen::Tensor<Scalar, Rank>::Tensor;  // inherit base constructors
+    using Eigen::Tensor<Scalar, _Rank>::Tensor;  // inherit base constructors
```

Comment:
```suggestion
    using Eigen::Tensor<Scalar, Rank>::Tensor;  // inherit base constructors
```

Reply:
underscores appear to be required to prevent the warnings.